client/pwa/src/db.ts

```diff
@@ -8,7 +8,6 @@ import Dexie from "dexie";
 export enum SwType {
   PreferOnline = "PreferOnline",
```

Comment:
It looks like this `enum` doesn't even belong inside here, because it isn't used by the Dexie database schema?

Reply:
This enum is shared between the service worker and the client. And we used to use this file to sync to make sure it's in sync. Something we should change once we decide on the fate of the service worker.